services/shoutouts.js

```diff
@@ -0,0 +1,34 @@
+const users = require('../data/dbModels/users');
+const shoutouts = require('../data/dbModels/shoutouts');
+module.exports = {
+  getShoutouts: async function(req, res, next) {
+    try {
+        const { userId } = req.params;
+        const user = await users.readBySlackId(userId);
+        if(!user || !user.slack_mem_id) {
+          return res.status(404)
```

Comment:
According to the article, we are following in designing our architecture, 
Don't pass the req or res object to the service layer
Don't return anything related to the HTTP transport layer like a status code or headers from the service layer.
Refer to this link https://dev.to/santypk4/bulletproof-node-js-project-architecture-4epf#service